entc/gen/template/ent.tmpl

```diff
@@ -28,7 +28,7 @@ type {{ $.Name }} struct {
 				// {{ $e.StructField }} holds the value of the {{ $e.Name }} edge.
 				{{ $e.StructField }} {{ if not $e.Unique }}[]{{ end }}*{{ $e.Type.Name }} {{ with $e.StructTag }}`{{ . }}`{{ end }}
 			{{- end }}
-		}
+		} `json:"edges,omitempty"`
```

Comment:
`omitempty` is irrelevant here, because `Edges` is a non-empty value (even if no edges were loaded).
```suggestion
		} `json:"edges"`
```

Reply:
Missed it. removed it.